implementations/java-openapiprocessor/src/main/kotlin/commands/Start.kt

```diff
@@ -0,0 +1,43 @@
+package commands
+
+import java.net.URI
+
+
+data class StartRequest(val version: Int): Request
+
+class Start(private val cfg: Configuration) {
+
+    fun run(request: StartRequest): Map<String, Any> {
+        assert(request.version == 1)
+
+        return mapOf(
+            "ready" to true,
+            "version" to 1,
+            "implementation" to mapOf(
+                "language" to "java",
+                "language_version" to "11",
```

Comment:
Can we not get this information slightly more specifically (e.g. I assume there's a minor version at least?) and dynamically? This is going to go out of date if we change the image.

Reply:
the java version provides the minimum version required to use it (the version it is compiled with). This is usually given just with the major version.

more correct would be to say something like `"language_version" to "11 or newer"` if it fits.

Reply:
This field is for Bowtie, it's meant to show users what version is the one being run -- e.g., imagine someone observes different behavior from what Bowtie reports than what they see themselves.

In that case they're not interested in the versions your library is saying it supports, they want to know exactly what's running so they can reproduce if needed.

Reply:
oh,  *that* version :-)